Classes/Services/MetsExporter.php

```diff
@@ -793,6 +795,16 @@ public function getSlubData()
     }
 
     /**
+     * replaces single quote with double quote
+     * @param $string
+     * @return string
+     */
+    function replaceSingleQuote($string)
```

Comment:
A static method does not require the (implicit) `this` function argument, so the code is smaller and faster.